### PR TITLE
Irminger 10 sub-surface telemetered

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

File: GI02HYPM/GI02HYPM_D00010_ingest.csv [hunk 1]
```diff
@@ -0,0 +1,8 @@
...
+mi.dataset.driver.sio_eng.sio.sio_eng_sio_telemetered_driver,/omc_data/whoi/OMC/GI02HYPM/D00010/node*p1_*.status*.dat,GI02HYPM-RIM01-00-SIOENG000,telemetered,Available,
+mi.dataset.driver.ctdmo_ghqr.sio.ctdmo_ghqr_sio_telemetered_driver,/omc_data/whoi/OMC/GI02HYPM/D00010/*ctdmo_*.dat,GI02HYPM-RIM01-02-CTDMOG039,telemetered,Available,
+mi.dataset.driver.WFP_ENG.wfp_sio.wfp_eng_wfp_sio_telemetered_driver,/omc_data/whoi/OMC/GI02HYPM/D00010/*.we_wfp_*.dat,GI02HYPM-WFP02-00-WFPENG000,telemetered,Available,
+mi.dataset.driver.flord_l_wfp.sio.flord_l_wfp_sio_telemetered_driver,/omc_data/whoi/OMC/GI02HYPM/D00010/*.we_wfp_*.dat,GI02HYPM-WFP02-01-FLORDL000,telemetered,Available,
+mi.dataset.driver.dosta_ln.wfp_sio.dosta_ln_wfp_sio_telemetered_driver,/omc_data/whoi/OMC/GI02HYPM/D00010/*.we_wfp_*.dat,GI02HYPM-WFP02-03-DOSTAL000,telemetered,Available,
+mi.dataset.driver.ctdpf_ckl.wfp_sio.ctdpf_ckl_wfp_sio_telemetered_driver,/omc_data/whoi/OMC/GI02HYPM/D00010/*.wc_wfp_*.dat,GI02HYPM-WFP02-04-CTDPFL000,telemetered,Available,
+mi.dataset.driver.vel3d_l.wfp.sio.vel3d_l_wfp_sio_telemetered_driver,/omc_data/whoi/OMC/GI02HYPM/D00010/*.wa_wfp_*.dat,GI02HYPM-WFP02-05-VEL3DL000,telemetered,Available,
```
Comment (rtravis-whoi) [inline GI02HYPM/GI02HYPM_D00010_ingest.csv:8]: looks good - data on raw data repo

File: GI03FLMA/GI03FLMA_D00010_ingest.csv [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+#,/omc_data/whoi/OMC/GI03FLMA/D00010/*ctdmo*.dat,GI03FLMA-RIM01-02-CTDMOH050,telemetered,Available,Data assigned to refdes after ingestion with wildcard refdes.No acoustic comms with FLMA modem
+#,/omc_data/whoi/OMC/GI03FLMA/D00010/*ctdmo*.dat,GI03FLMA-RIM01-02-CTDMOH051,telemetered,Available,Data assigned to refdes after ingestion with wildcard refdes.No acoustic comms with FLMA modem
+mi.dataset.driver.sio_eng.sio.sio_eng_sio_telemetered_driver,/omc_data/whoi/OMC/GI03FLMA/D00010/*status_1328301.dat,GI03FLMA-RIS01-00-SIOENG000,telemetered,Available,
+mi.dataset.driver.dosta_abcdjm.sio.dosta_abcdjm_sio_telemetered_driver,/omc_data/whoi/OMC/GI03FLMA/D00010/*dosta*.dat,GI03FLMA-RIS01-03-DOSTAD000,telemetered,Available,
+mi.dataset.driver.phsen_abcdef.sio.phsen_abcdef_sio_telemetered_driver,/omc_data/whoi/OMC/GI03FLMA/D00010/*phsen*.dat,GI03FLMA-RIS01-04-PHSENF000,telemetered,Available,
+mi.dataset.driver.flort_dj.sio.flort_dj_sio_telemetered_driver,/omc_data/whoi/OMC/GI03FLMA/D00010/*flort*.dat,GI03FLMA-RIS01-05-FLORTD000,telemetered,Available,
```
Comment (rtravis-whoi) [inline GI03FLMA/GI03FLMA_D00010_ingest.csv:20]: looks good - data available on raw data repo

**Review by rtravis-whoi — APPROVED** [review]

File: GI02HYPM/GI02HYPM_D00010_ingest.csv (hunk 1 above)
Comment (johnmlund) [inline GI02HYPM/GI02HYPM_D00010_ingest.csv:8]: looks good

File: GI03FLMA/GI03FLMA_D00010_ingest.csv (hunk 2 above)
Comment (johnmlund) [inline GI03FLMA/GI03FLMA_D00010_ingest.csv:20]: looks good

File: GI03FLMB/GI03FLMB_D00010_ingest.csv [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+#,/omc_data/whoi/OMC/GI03FLMB/D00010/*ctdmo*.dat,GI03FLMB-RIM01-02-CTDMOH069,telemetered,Available,Data assigned to refdes after ingestion with wildcard refdes.
+#,/omc_data/whoi/OMC/GI03FLMB/D00010/*ctdmo*.dat,GI03FLMB-RIM01-02-CTDMOH070,telemetered,Available,Data assigned to refdes after ingestion with wildcard refdes.
+#,/omc_data/whoi/OMC/GI03FLMB/D00010/*ctdmo*.dat,GI03FLMB-RIM01-02-CTDMOH071,telemetered,Available,Data assigned to refdes after ingestion with wildcard refdes.
+mi.dataset.driver.sio_eng.sio.sio_eng_sio_telemetered_driver,/omc_data/whoi/OMC/GI03FLMB/D00010/*status_1398901.dat,GI03FLMB-RIS01-00-SIOENG000,telemetered,Available,
+mi.dataset.driver.dosta_abcdjm.sio.dosta_abcdjm_sio_telemetered_driver,/omc_data/whoi/OMC/GI03FLMB/D00010/*dosta*.dat,GI03FLMB-RIS01-03-DOSTAD000,telemetered,Available,
+mi.dataset.driver.phsen_abcdef.sio.phsen_abcdef_sio_telemetered_driver,/omc_data/whoi/OMC/GI03FLMB/D00010/*phsen*.dat,GI03FLMB-RIS01-04-PHSENF000,telemetered,Available,
+mi.dataset.driver.flort_dj.sio.flort_dj_sio_telemetered_driver,/omc_data/whoi/OMC/GI03FLMB/D00010/*flort*.dat,GI03FLMB-RIS01-05-FLORTD000,telemetered,Available,
```
Comment (johnmlund) [inline GI03FLMB/GI03FLMB_D00010_ingest.csv:20]: looks good

**Review by johnmlund — APPROVED** [review]

**Pull request merged**